client/asset/dcr/dcr.go

```diff
@@ -391,7 +391,7 @@ func (dcr *ExchangeWallet) Fund(value uint64, nfo *dex.Asset) (asset.Coins, erro
 	enough := func(sum uint64, size uint32, unspent *compositeUTXO) bool {
 		return sum+toAtoms(unspent.rpc.Amount) >= dcr.reqFunds(value, size+unspent.input.Size(), nfo)
 	}
-	coins, _, _, err := dcr.fund(nfo.FundConf, enough)
+	coins, _, _, err := dcr.fund(0, enough)
```

Comment:
The confirmation argument could be removed if this change sticks forever, but it doesn't hurt to remain.